src/dialects/abstract/query-interface.js

```diff
@@ -459,7 +462,12 @@ export class QueryInterface {
    * @param {object} [options]      Query options
    */
   async removeColumn(tableName, attributeName, options) {
```

Comment:
```suggestion
  async removeColumn(tableName, attributeName, options = {}) {
```
This might be a nice refactor, for `addColumn` as well I think. Or is there a reason not to do this?

Reply:
The same as above, also lemme give a reason as well why am I doing that `null`. The options is directly passed in the `rejectInvalidOptions` so as it skip it when no args are present I pass null.

And for double checking I am doing this.

```js
options = options || {};
```

Reply:
Maybe I'm missing something, but don't they do the same? So I want to set it by default when defining the function and you want to add it two lines later on.
In both cases `options` will be `{}` if it has not been set but in my suggestion it is done in the definition directly instead of having an additional line of code

Reply:
> const options = ifExists ? { ifExists } : null;

Prefer setting it to undefined, so the default parameter operator can be used. I think it's cleaner